src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+          schema:
+            type : string 
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: IntrospectionResponse
+                $ref: '#/components/schemas/IntrospectionResponse'                                        
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'  
+  /.jwks:
+    get:
+      tags:
+      - developers
+      summary: A JSON Web Key (JWK) Endpoint to get set of public keys as a JSON object.  It is sometimes useful to be able to reference public key representations, for instance, in order to verify the signature on content signed with the corresponding private key. The JSON Web Key (JWK) data structure provides a convenient JSON representation for sets of public keys utilizing either the Elliptic Curve or RSA families of algorithms.
+      description: A JSON Web Key (JWK) is a JSON data structure that represents a set of public keys as a JSON object.
+      operationId: requestJwk      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: WebKeysConfiguration
+                required:
+                - keys
+                type: object
+                properties:
+                  keys:
+                    type: object
+                    additionalProperties:
+                      type: object                                    
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'   
+  /register:
+    post:
+      tags:
+      - developers
+      summary: The Client Registration Endpoint is an OAuth 2.0 Protected Resource through which a new Client registration can be requested.
+      description: Registers new client dynamically.
+      operationId: requestRegister
+      requestBody:
+        content:
+          application/json:
+            schema:
+              title: RegisterParams
+              required:
+              - redirect_uris
+              type: object
+              properties:
+                redirect_uris:
+                  type: array
+                  description: Request parameters as JSON object with data described by Connect Client Registration Specification.
+                  example:
+                  - https://client.example.org/cb
+                  items:
+                    type: string
```

Comment:
I guess the whole story of swagger begins that we want to list accurately all parameters for Dynamic Registration. Here I can see only `redirect_uris`. I'm sure there are more parameter .